openapi2beans/pkg/generator/yaml2java.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package generator
+
+import (
+	"log"
+	"strings"
+
+	openapi2beans_errors "github.com/dev-galasa/buildutils/openapi2beans/pkg/errors"
+	"github.com/galasa-dev/cli/pkg/files"
+)
+
+func GenerateFiles(fs files.FileSystem, projectFilePath string, apiFilePath string, packageName string) error {
+	var fatalErr error
+	var apiyaml string
+	var errList map[string]error
+
+	storeFilePath := generateStoreFilePath(projectFilePath, packageName)
+	fatalErr = generateDirectories(fs, storeFilePath)
+	if fatalErr == nil {
+		apiyaml, fatalErr = fs.ReadTextFile(apiFilePath)
+		if fatalErr == nil {
+			var schemaTypes map[string]*SchemaType
+			schemaTypes, errList, fatalErr = getSchemaTypesFromYaml([]byte(apiyaml))
+			if fatalErr == nil {
+				javaPackage := translateSchemaTypesToJavaPackage(schemaTypes, packageName)
```

Comment:
No error passed back from this method, so errors won't stop it generating files.

Reply:
hmm, will have it not generate anything if something fails then.